ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteBucket.java

```diff
@@ -56,6 +56,17 @@ public CliCommand init(final Arguments args) throws Exception {
     }
 
     @Override
+    public String getLongHelp() {
+        final StringBuffer helpStringBuffer = new StringBuffer();
```

Comment:
With newer version of java you do not need to use a StringBuffer when concatenating strings together, if all the strings are constants.  The compiler will optimize it for you.  We should also think about making these into resource files so we can send them to tech pups for editing.